apm-agent-plugins/apm-vertx/apm-vertx4-plugin/src/main/java/co/elastic/apm/agent/vertx/v4/webclient/HttpContextInstrumentation.java

```diff
@@ -122,7 +122,13 @@ public static void sendRequest(@Advice.This HttpContext<?> httpContext, @Advice.
 
                 if (parentSpan != null) {
                     AbstractSpan<?> parent = (AbstractSpan<?>) parentSpan;
-                    webClientHelper.startSpan(parent, httpContext, request);
+                    //TODO: do actual context propagation instead of span propagation
```

Comment:
is that something you plan to do in the second PR ?

Reply:
Yes. It won't be in the next PR, but in the one after where I adapt all of our intra-process context propagations.
I did this after the addition of baggage so that I could actually add tests for it.